modules/op-creating-webhooks.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="creating-webhooks_{context}"]
+= Creating Webhooks
+
+_Webhooks_ are HTTP POST messages that are received by the EventListeners whenever a configured event occurs in your repository. The event payload is then mapped to TriggerBindings, and processed by TriggerTemplates which eventually starts a PipelineRun(s) leading to the creation and deployment of Kubernetes resources.
+
+In this section you will configure a Webhook URL on your forked Git repositories--front-end `vote-ui` and back-end `vote-api`. This URL points to the EventListener service route which you made publicly accessible in the previous step.
+
+[NOTE]
+====
+Adding Webhooks requires administrative privileges to the repository. If you do not have administrative access to your repository, contact your System Administrator for adding Webhooks.
+====
+
+[discrete]
+.Procedure
+
+. Get the Webhook URL:
++
+----
+$ echo "URL: $(oc  get route el-vote-app --template='http://{{.spec.host}}')"
+----
++
+Note the URL obtained in the output.
+
+. Configure Webhooks manually on front-end repository:
+
+.. Open the front-end Git repository `vote-ui` in your browser.
+.. Click *Settings* and then *Webhooks*.
+.. Click *Add Webhook*
```

Comment:
Suggest: Navigate to *Settings* -> *Webhooks* -> *Add Webhook*. instead of step ii and iii